src/mudlet-lua/lua/Other.lua

```diff
@@ -832,3 +845,51 @@ do
     end
   end
 end
+
+
+-- A utility function based on tempTimers meant to track all sorts of temporary states
+local timeframeTable = {}  -- keeps track of all timeframe timers
+function timeframe(vname, true_time, nil_time, ...)
+  local timerlist = {
+    { 0 }, -- reset variable 'vname' to nil
```

Comment:
As discussed on discord, I am still debating whether we actually want to reset `vname` or not... If it's set anyways, why would we want to?

My personal opinion is that we don't want to reset it, but that's not set in stone.